assets/app/view/create_game.rb

```diff
@@ -272,6 +279,8 @@ def update_inputs
         store(:num_players, range.value.to_i)
       end
 
+      store(:selected_game, selected_game)
```

Comment:
store rerenders the view unless you do skip: true, so change this one and the store(:num_players to have skip: true

Reply:
Done, thanks.